src/tools/chip-cert/CertUtils.cpp

```diff
@@ -142,12 +142,12 @@ void ToolChipDN::PrintDN(FILE * file, const char * name) const
         }
         else
         {
-            uint32_t len = rdn[i].mAttrValue.mString.mLen;
+            uint32_t len = static_cast<uint32_t>(rdn[i].mAttrValue.mString.size());
```

Comment:
Shouldn't `len` just be `size_t` or `auto` here?

Reply:
done